packages/ethereum-viewer/src/extension.ts

```diff
@@ -49,7 +50,7 @@ async function detectExplorerApiName(): Promise<explorer.ApiName> {
   if (IN_DETH_HOST) {
     const detectedName = await executeHostCommand("getApiName");
 
-    if (detectedName === null) return "etherscan";
+    if (!detectedName) return "etherscan";
```

Comment:
technically it's not the same as ` === undefined` but in this case the difference shouldn't make any difference

Reply:
Yeah, even if it's an empty string, we don't care, we'll rather use `"etherscan"`.